### ISSUE TITLE
Should I pull the develop branch before starting a new feature ?

### ISSUE THREAD

**Issue by faressoft** [danielkummer/git-flow-cheatsheet#75]
Should I pull the develop branch before starting a new feature ?

**Comment by danielkummer** [comment]
Definitely! You should always start with the most current develop branch to
mitigate merging issues...

**Comment by MrCsabaToth** [comment]
I don't see any development branch. If I translate it to Hungarian based on a fork of the gh-pages branch, will the PR dry out like the Azerbajani translation PR?

**Comment by danielkummer** [comment]
Sorry, it's not the develop branch but directly the gh-pages branch.

And no, it won't dry out - I'm having time on my hands again to support the PRs

**Comment by juliojgd** [comment]
I did a `git flow init` over the repo. I am beginning with Galician translation. Is it intended to use git-flow here? Or directly push to `gh-pages`?
